signing/sign.sh

```diff
@@ -56,7 +56,10 @@ fero-client \
 	--secret-key coreos-app-signing-key \
 	${torcx_signature_arg}
 gpg2 --enarmor \
-	--output "${DATA_DIR}/torcx_manifest.json.asc" \
-	"${DATA_DIR}/torcx_manifest.json.sig-fero"
+	--output - \
+	"${DATA_DIR}/torcx_manifest.json.sig-fero" \
+	| sed 's/ARMORED FILE/SIGNATURE/' \
```

Comment:
Ridiculous nit: `ARMORED FILE` could theoretically show up in the signature body, but `-` can't, so probably better to check for the trailing `-----`.

Reply:
Are spaces allowed? I don't think they are.

Reply:
Ah, you're right.